development/metamaskbot-build-announce.js

```diff
@@ -87,6 +87,9 @@ async function start() {
     .map((key) => `<li>${key}: ${bundles[key].join(', ')}</li>`)
     .join('')}</ul>`;
 
+  const bundleSizeDataUrl =
+    'https://raw.githubusercontent.com/MetaMask/extension_bundlesize_stats/main/stats/bundle_size_data.js';
```

Comment:
Hmm. This could be misleading. We don't require branches to be up-to-date before merging in this repository, so sometimes the parent commit for a branch is an older `develop` commit. But this is assuming that the most recent bundle size stats commit is what we should compare with.

This could result in false results. For example, if the most recent commit on `develop` reduced the bundle size, this would wrongly credit the current PR for that size reduction.

I'm not sure how we could get a reference to the correct commit though :thinking:

Reply:
Maybe we can start tagging commits in the `extension_bundlesize_stats` repo with the extension commit hash? Then we could reference it by tag

Reply:
Oh, no need, they're already available as files with the commit hash in the filename,

e.g. `bundle_size_stats-0026966c9ea5b15fe99b83415b8c90334c87877f.json`

Reply:
@Gudahtt seems like we don't commit the stats to a specific per-commit file anymore, now `bundle_size_data.js` should hold data for every commit on main. 

I guess I should extract the right commit from `bundle_size_data.js` with a regex (as it's not a json) considering the develop commit from which the pr branch is starting 
- Do you know if we have that commit sha in circleci env vars? I took a look at docs but wasn't able to find any. If not, I should find the var in the circle pipeline and pass it to the js script

Reply:
Thanks @Gudahtt! I created [this PR](https://github.com/MetaMask/metamask-extension/pull/16168) to add a JSON version of the file and update it along with the JS one.
I created a different PR so we can eventually merge it before this one.

Reply:
Great, thanks! Good idea to handle it separately